apps/api/src/server/applications/application/documents/document.controller.js

```diff
@@ -30,21 +32,25 @@ import {
 	validateDocumentVersionMetadataBody,
 	verifyAllDocumentsHaveRequiredPropertiesForPublishing
 } from './document.validators.js';
-import { mapDateStringToUnixTimestamp } from '../../../utils/mapping/map-date-string-to-unix-timestamp.js';
 
 /**
- * @typedef {import('apps/api/src/database/schema.js').Document} Document
- * @typedef {import('apps/api/src/database/schema.js').DocumentDetails} DocumentDetails
- * @typedef {import('apps/api/src/database/schema.js').DocumentVersionInput} DocumentVersion
+ * @typedef {import('@prisma/client').Document} Document
+ * @typedef {import('@prisma/client').DocumentVersion} DocumentVersion
+ * @typedef {import('@pins/applications.api').Schema.DocumentDetails} DocumentDetails
+ * @typedef {import('@pins/applications.api').Schema.DocumentVersionWithDocument} DocumentVersionWithDocument
+ * @typedef {import('@pins/applications.api').Api.DocumentToSave} DocumentToSave
+ * @typedef {import('@pins/applications.api').Api.DocumentToSaveExtended} DocumentToSaveExtended
+ * @typedef {import('@pins/applications.api').Api.DocumentsToSaveManyRequestBody} DocumentsToSaveManyRequestBody
  */
 
 /**
- * @type {import('express').RequestHandler<any, any, { blobStorageHost: string, privateBlobContainer: string, documents: { documentName: string, blobStoreUrl: string }[] } | any, any>}
+ * Upload an array of documents to a folder on a case
+ *
+ * @type {import('express').RequestHandler<any, any, { DocumentsToSaveManyRequestBody } | any, any>}
```

Comment:
I think this is meant to be `DocumentsToSaveManyRequestBody | any` (I'm assuming here the body is meant to be an array).